app/controllers/InitialiseAppealController.scala

```diff
@@ -16,28 +16,29 @@
 
 package controllers
 
+import java.time.LocalDate
+import java.util.UUID
+
+import _root_.config.featureSwitches.{EnablePRM2509, FeatureSwitching}
```

Comment:
root?!?!!?!?

Not sure why its prompted you to use root but this should work?
```suggestion
import config.featureSwitches.{EnablePRM2509, FeatureSwitching}
```

Reply:
Probably that command of yours....I definitely didn't manually add that haha